packages/cra-template-defencedigital/template.json

```diff
@@ -18,7 +18,7 @@
       "@graphql-codegen/typescript-react-apollo": "^3.2.2",
       "@juggle/resize-observer": "^3.3.1",
       "@testing-library/jest-dom": "^5.5.0",
-      "@testing-library/react": "^12.1.2",
+      "@testing-library/react": "^13.1.1",
       "@testing-library/user-event": "^13.5.0",
       "@types/jest": "^27.0.3",
       "@types/node": "^16.11.13",
```

Comment:
Would be good to update `@types/react` and `@types/react-dom` below to 18.x versions as well (to match the version of React installed)

Reply:
Nice spot!